src/engine.rs

```diff
@@ -241,7 +242,11 @@ impl Engine for DevmodeEngine {
                             break;
                         }
                         Update::BlockNew(block) => {
-                            info!("Checking consensus data: {:?}", block);
+
+                            if &block.previous_id == &NULL_BLOCK_IDENTIFIER {
```

Comment:
Not a blocking change request, but this might be easier and more obvious if you just check `block.block_num == 0`

Reply:
Based on some offline conversations recently, I figured it would be good to get away from the use of block num, and use the correct method for ordering, which is the previous id.